npc/re/jobs/2e/rebellion.txt

```diff
@@ -0,0 +1,1082 @@
+//================= Hercules Script =======================================
+//=       _   _                     _
+//=      | | | |                   | |
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2020 Hercules Dev Team
+//= Copyright (C) Dastgir
+//=
+//= Hercules is free software: you can redistribute it and/or modify
+//= it under the terms of the GNU General Public License as published by
+//= the Free Software Foundation, either version 3 of the License, or
+//= (at your option) any later version.
+//=
+//= This program is distributed in the hope that it will be useful,
+//= but WITHOUT ANY WARRANTY; without even the implied warranty of
+//= MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//= GNU General Public License for more details.
+//=
+//= You should have received a copy of the GNU General Public License
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Rebellion Job Quest
+//================= Description ===========================================
+//= Job change Quest from Gunslinger -> Rebellion
+//================= Current Version =======================================
+//= 1.0
+//=========================================================================
+
+job_gun,1,1,1	script	Administrator1	CLEAR_NPC,{
+	callfunc("F_GM_NPC");
+	mes("Please enter the password.");
+	.@success = callfunc("F_GM_NPC", 1854, 0);
+	if (.@sucess != 1)
+		close;
+
+	mes("[Time Administrator]");
+	mes("What time would you like to return?");
+	next;
+	.@select = select("Cancel", "Test monster spawn", "Start from beginning", "Furnace Zone", "Shooting Zone", "Complete the test");
+	if (.@select == 1) {
+		close;
+	} else if (.@select == 2) {
+		donpcevent("#target::OnStart");
+		donpcevent("#nontarget::OnStart");
+		close;
+	} else if (.@select == 4) {
+		.@questComplete = 12341;
+	} else if (.@select == 5) {
+		.@questComplete = 12342;
+	} else if (.@select == 6) {
+		.@questComplete = 12344;
+	}
+	for (.@i = 12340; .@i <= 12345; .@i++) {
+		erasequest(.@i);
+		if (.@i <= .@questComplete) {
+			setquest(.@i);
+			completequest(.@i);
+		}
+	}
+	close;
+
+OnInit:
+	disablenpc(strnpcinfo(NPC_NAME));
+	end;
+}
+
+job_gun,56,26,0	script	rebelroom#1	WARPNPC,2,2,{
```

Comment:
Please convert these to one line style warps

```
job_gun,56,26,0	warp	rebelroom#1	2,2,einbroch,49,97
```

Reply:
Done